game/scripts/npc/abilities/monkey_king_wukongs_command.txt

```diff
@@ -20,7 +20,7 @@
 
 		// Time		
 		//-------------------------------------------------------------------------------------------------------------
-		"AbilityCooldown"				"100 85 70 70 70"
+		"AbilityCooldown"				"130 110 90 70 70"
```

Comment:
`130 110 90 90 90` The later levels scale largely with hero power, so we don't want it to go crazy